webauthn.cabal

```diff
@@ -96,6 +96,7 @@ library
     serialise             >= 0.2.3 && < 0.3,
     singletons            >= 2.6 && < 3.2,
     text                  >= 1.2.4 && < 2.1,
+    these                 >= 1 && < 2,
```

Comment:
```suggestion
    these                 >= 1.1 && < 1.2,
```

Reply:
I've kept upper bounds to 1.3 to align with the other packages that cover multiple minor versions.